go/parquet/internal/encoding/delta_bit_packing.go

```diff
@@ -466,6 +466,10 @@ func (enc *deltaBitPackEncoder) FlushValues() (Buffer, error) {
 
 // EstimatedDataEncodedSize returns the current amount of data actually flushed out and written
 func (enc *deltaBitPackEncoder) EstimatedDataEncodedSize() int64 {
+	if enc.bitWriter == nil {
```

Comment:
C++ version uses the syntax similiar with `enc.sink.Length()`, but maybe it's more elegant here